admin/test/test_release.py

```diff
@@ -903,6 +903,8 @@ def test_fake_rpm(self):
                          '<oldhash>-metadata.xml'):
                 'metadata for: existing_package.rpm',
         }
+        # Copy before passing to FakeAWS
+        expected_keys = existing_s3_keys.copy()
```

Comment:
As this was a fix for an existing bug in the tests, unrelated to the changes done for this PR, I decided not to change the implementation code.